src/main/webui/src/app/components/content/group/GroupEdit.jsx

```diff
@@ -14,8 +14,214 @@
+            }
+          </div>
+
+          <div className="detail-field">
+            <input
+              type="checkbox"
+              defaultChecked={!store.disabled}
+              {...register("enabled")}
+            />{" "}
+            <label>Enabled?</label>
+            {store.disabled && store.disableExpiration &&
+              <span>
+                <Hint hintKey="Set to automatically re-enable at {TimeUtils.timestampToDateFormat(store.disableExpiration)}" />
+              </span>
+            }
+          </div>
+          <div className="detail-field">
+            <input
+              type="checkbox"
+              defaultChecked={store.prepend_constituent}
+              {...register("prepend_constituent")}
+            />{" "}
+            <label>Prepend Constituents?</label>
+              <span>
+                <Hint hintKey="If enabled, all new constituents which are added not manually(like promotion) will be at the top of constituents list" />
+              </span>
+          </div>
+
+          <div className="sub-fields">
+            <div className="detail-field">
+              <label>Disable Timeout:</label>
+              <input
+                type="number"
+                defaultValue={store.disable_timeout}
+                {...register("disable_timeout", {min: -1, max: 999999999})}
+              />{" "}
+              {errors.disable_timeout &&
+                <span className="alert">Not a valid number</span>
+              }
+              <br />
+              <DisableTimeoutHint />
+            </div>
+          </div>
+        </div>
+
+        <div className="fieldset-caption">Description</div>
+        <div className="fieldset">
+          <textarea
+            rows="3"
+            className="text-description"
+            {...register("description")}
+          >
+            {store.description}
+          </textarea>
+        </div>
+
+        <div className="fieldset-caption">Constituents</div>
+        <div className="fieldset">
+          {
+                store.constituents && store.constituents.length > 0 &&
+                <ol className="left-col detail-value detail-edit-list">
+                  {
+                    store.constituents.map(item => <li key={`constituent-${item}`} className="detail-edit-list-item">
+                      <Link to={Utils.detailHref(item)}>{item}</Link>
+                    </li>)
+                  }
+                </ol>
+              }
+          {
+                store.available && store.available.length > 0 &&
+                <ol className="right-col detail-value detail-edit-list">
+                  {
+                    store.available.map(item => <li key={`available-${item}`} className="detail-edit-list-item">
+                      <Link to={Utils.detailHref(item)}>{item}</Link>
+                    </li>)
+                  }
+                </ol>
+              }
+        </div>
+      </div>
```

Comment:
The constituents part is not just showing the all item as link, but also has abilities to:

* Add item from available to current
* Delete item in current to available
* Promote/demote priority(order) of item in current, or do top/bottom function in current.

Reply:
removed group-edit part from this PR